clients/pkg/promtail/targets/kafka/parser.go

```diff
@@ -0,0 +1,25 @@
+package kafka
+
+import (
+	"github.com/Shopify/sarama"
+	"github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/model/relabel"
+
+	"github.com/grafana/loki/clients/pkg/promtail/api"
+	"github.com/grafana/loki/pkg/logproto"
+)
+
+// noopMessageParser implements MessageParser. It doesn't modify the content of the original `message.Value`.
+type noopMessageParser struct{}
```

Comment:
I wasn't expecting this by its name; I mainly was expecting it to be some kind of test helper or something. Do we want to make it into `kafkaMessageParser` or something?